src/hooks/use-authentication/helpers/make-sign-in.ts

```diff
@@ -15,14 +10,14 @@ type SignInOptions = Pick<_SignInOptions, 'callbackUrl' | 'redirect'>
  *
  * https://next-auth.js.org/getting-started/client#signin
  */
-const makeSignIn = ({ routerPath }: MakeSignInOptions) => {
+const makeSignIn = () => {
 	return (
 		provider: ValidAuthProviderId = DEFAULT_PROVIDER_ID,
 		options: SignInOptions = {}
 	) => {
-		const { callbackUrl = routerPath, redirect = true } = options
+		const { redirect = true } = options
 
-		return signIn(provider, { callbackUrl, redirect })
+		return signIn(provider, { redirect })
```

Comment:
The default behavior is to redirect to the page that initiated sign in, I don't think this will fix the issue but a small thing to adjust to simplify